shared/src/bundles/users/enums/users-api-path.enum.ts

```diff
@@ -1,5 +1,6 @@
 const UsersApiPath = {
     ROOT: '/',
+    CURRENT: '/current-user',
```

Comment:
I think there is no need to duplicate user in path here

Reply:
The root path segment is "users" so your version may suggest that it will return a list when in fact it returns a single user, so I think current-user is more descriptive

Reply:
No, I meant for example users/current, that means current user